### PR TITLE
[IGOWEB-1595] Show all erros for cmo id generation

### PR DESCRIPTION
**Pull request by imprezka88**

### CODE REVIEW

File: src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/SampleTypeCorrectedCmoSampleIdGenerator.java [hunk 1]
```diff
@@ -150,4 +152,8 @@ private void notifyAboutCorrectedCmoIdFailure(CorrectedCmoSampleView correctedCm
                     " sample: %s", correctedCmoSampleView.getId()));
         }
     }
+
+    private void validate(List<CorrectedCmoSampleView> views) {
```
Comment (lvncnt) [inline src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/SampleTypeCorrectedCmoSampleIdGenerator.java]: `validate` logic is left empty intentionally?

File: src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/converter/SampleToCorrectedCmoIdConverter.java [hunk 2]
```diff
@@ -49,8 +52,33 @@ public CorrectedCmoSampleView convert(Sample sample) throws LimsException {
...
+        error = appendToError(sample, s -> (SampleType) SampleType.fromString(sample.getExemplarSampleType()), error);
+        error = appendToError(sample, s -> (Recipe) Recipe.getRecipeByValue(sample.get(Sample.RECIPE)), error);
+        error = appendToError(sample, s -> (SampleClass) SampleClass.fromValue(sample.getCorrectedSampleClass()),
+                error);
+        error = appendToError(sample, s -> (SampleOrigin) SampleOrigin.fromValue(sample.getCorrectedCmoSampleOrigin()
+        ), error);
+        error = appendToError(sample, s -> (SpecimenType) SpecimenType.fromValue(sample.getCorrectedSpecimenType()),
+                error);
+
+        throw new RuntimeException(error.toString());
```
Comment (lvncnt) [inline src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/converter/SampleToCorrectedCmoIdConverter.java]: if `error ` is empty, exception will be thrown anyway?

File: src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/SampleTypeCorrectedCmoSampleIdGenerator.java (hunk 1 above)
Comment (imprezka88) [reply src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/SampleTypeCorrectedCmoSampleIdGenerator.java]: Nope, I forgot to delete it. thanks!

File: src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/converter/SampleToCorrectedCmoIdConverter.java (hunk 2 above)
Comment (imprezka88) [reply src/main/java/org/mskcc/limsrest/limsapi/cmoinfo/converter/SampleToCorrectedCmoIdConverter.java]: Yep, I changed it and will commit changes

**Pull request merged**